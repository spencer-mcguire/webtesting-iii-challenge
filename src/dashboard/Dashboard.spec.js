import React from "react";
import { render } from "@testing-library/react";
import Dashboard from "./Dashboard";

test("FULL dashboard renders without crashing", () => {
  render(<Dashboard />);
});

test("Matches Snapshot", () => {
  const { baseElement } = render(<Dashboard />);
  expect(baseElement).toMatchSnapshot();
});
