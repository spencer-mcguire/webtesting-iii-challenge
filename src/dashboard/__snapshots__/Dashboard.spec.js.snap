// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Snapshot 1`] = `
<body>
  <div>
    <div
      class="display panel"
      data-testid="mainDisplay"
    >
      <div
        class="led green-led"
        data-testid="locked-display"
      >
        Unlocked
      </div>
      <div
        class="led green-led"
        data-testid="closed-display"
      >
        Open
      </div>
    </div>
    <div
      class="controls panel"
      data-testid="mainControls"
    >
      <button
        class="toggle-btn"
        disabled=""
      >
        Lock Gate
      </button>
      <button
        class="toggle-btn"
      >
        Close Gate
      </button>
    </div>
  </div>
</body>
`;
